core/src/main/java/org/apache/accumulo/core/iterators/OrIterator.java

```diff
@@ -80,6 +80,19 @@ public int compareTo(TermSource o) {
       // sorted after they have been determined to be valid.
       return this.iter.getTopKey().compareColumnQualifier(o.iter.getTopKey().getColumnQualifier());
     }
+
+    @Override
+    public void close() throws Exception {
+      this.iter.close();
+    }
+
+    public void closeSafely() {
```

Comment:
I made TermSource just implement AutoCloseable.  But I could make it implement SKVI instead